### PR TITLE
add guide for Angular

### PR DESCRIPTION
**Pull request by Quramy**
Issue:
There is no guide for Angular.

## What I did
Write angular-guide.

### CODE REVIEW

File: docs/src/pages/basics/guide-angular/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## title: 'Storybook for Angular'
+
+You may have tried to use our quick start guide to setup your project for Storybook. If you want to set up Storybook manually, this is the guide for you.
+
+> This will also help you understand how Storybook works.
+
+## Starter Guide Angular
+
+Storybook has its own Webpack setup and a dev server.
```
Comment (abouthiroppy) [inline docs/src/pages/basics/guide-angular/index.md]: An initial letter should be a lowercase letter;)
see: https://github.com/webpack/media#name

**Comment by kazupon** [comment]
LGTM 👍 
/ping @alterx

**Comment by alterx** [comment]
LGTM, my only concern is that is dones't touch on any of the specifics but as a quick guide I think it shouldn't

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
Thank you @Quramy !! 🙇

**Pull request merged**